pygpcca/_gpcca.py

```diff
@@ -1167,12 +1207,12 @@ def metastable_sets(self) -> Optional[List[np.ndarray]]:
 
         Returns
         -------
-        A list of length equal to :attr:`n_metastable`.
+        A list of length equal to :attr:`n_macrostates`.
 
         Each element is an array with microstate indexes contained in it.
         """
         return (
             None
-            if self.metastable_assignment is None or self.n_metastable is None
-            else [np.where(self.metastable_assignment == i)[0] for i in range(self.n_metastable)]
+            if self.metastable_assignment is None or self.n_macrostates is None
```

Comment:
I would either go for `metastable` of `macro` throughout the entire implementation.

Reply:
True.
Point is that metastable dosen't match for GPCCA (as it did for PCCA+), since we aren't restricted to finding metastable states, but dominant ones.
Thus I would simply call them macrostates.
Will check for consistency here...

Reply:
Done [here](https://github.com/msmdev/pyGPCCA/pull/13/commits/30d3f6d7afee3cc9d829c3f1a7a2a706eb098707) (but with some typos) and fixed [here](https://github.com/msmdev/pyGPCCA/pull/13/commits/980a5a6a32c2a2fdc23941454f59c4f042baa20f).

Reply:
awesome, nice.